<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuJuego</class>
 <widget class="QWidget" name="MenuJuego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #1a237e, stop:1 #283593);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameInfo">
     <property name="styleSheet">
      <string notr="true">background: rgba(255,255,255,0.2);
border: 2px solid rgba(255,255,255,0.3);
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblJugadores">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>JUGADOR ACTUAL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblJugadorActual">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>Jugador 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTurnos">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>TURNO:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTurno">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lblTamTablero">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>TABLERO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTamanioTablero">
        <property name="styleSheet">
         <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
        </property>
        <property name="text">
         <string>63</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameTablero">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #4caf50, stop:1 #388e3c);
border: 3px solid #2e7d32;
border-radius: 15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFrame" name="frameDado">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(255,255,255,0.15);
border: 2px solid rgba(255,255,255,0.3);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="btnDado">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">#btnDado {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #ff9800, stop:1 #f57c00);
    border: 3px solid #e65100;
    border-radius: 15px;
    color: white;
    font-family: 'Arial';
    font-size: 20px;
    font-weight: bold;
    padding: 20px;
}
#btnDado:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f57c00, stop:1 #ff9800);
}
#btnDado:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #e65100, stop:1 #f57c00);
}</string>
           </property>
           <property name="text">
            <string>🎲 LANZAR DADO 🎲</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblResultadoDado">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 16px; font-weight: bold; text-align: center;</string>
           </property>
           <property name="text">
            <string>Resultado: -</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameHistorial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background: rgba(255,255,255,0.15);
border: 2px solid rgba(255,255,255,0.3);
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblHistorial">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: white; font-size: 16px; font-weight: bold;</string>
           </property>
           <property name="text">
            <string>HISTORIAL:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txtHistorial">
           <property name="styleSheet">
            <string notr="true">QTextEdit {
    background: white;
	color: black;
    border: 2px solid #bdc3c7;
    border-radius: 5px;
    font-family: 'Courier New';
    font-size: 12px;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameBotones">
     <property name="styleSheet">
      <string notr="true">background: rgba(0,0,0,0.3); border-top: 1px solid rgba(255,255,255,0.2);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnGuardar">
        <property name="styleSheet">
         <string notr="true">#btnGuardar {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #2196f3, stop:1 #1976d2);
    border: 2px solid #1976d2;
    border-radius: 8px;
    color: white;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 15px;
    min-width: 100px;
}
#btnGuardar:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1976d2, stop:1 #2196f3);
}</string>
        </property>
        <property name="text">
         <string>💾 GUARDAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCargar">
        <property name="styleSheet">
         <string notr="true">#btnCargar {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #4caf50, stop:1 #388e3c);
    border: 2px solid #388e3c;
	border-radius: 8px;
    color: white;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 15px;
    min-width: 100px;
}
#btnGuardar:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1976d2, stop:1 #2196f3);
}</string>
        </property>
        <property name="text">
         <string>📂 CARGAR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMenuPrincipal">
        <property name="styleSheet">
         <string notr="true">#btnMenuPrincipal {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #f44336, stop:1 #d32f2f);
    border: 2px solid #d32f2f;
	border-radius: 8px;
    color: white;
    font-size: 14px;
    font-weight: bold;
    padding: 10px 15px;
    min-width: 100px;
}
#btnGuardar:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #1976d2, stop:1 #2196f3);
}</string>
        </property>
        <property name="text">
         <string>🏠 MENÚ PRINCIPAL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
